### ISSUE TITLE
use_stitch problems

### ISSUE THREAD

**Issue by Wendyuf** [limbo018/OpenMPL#16]
When I am doing a layout decomposition which is  from the small dataset, I can't get the result if I set the use_stitch command, it seems to stuck somewhere or how long does this work takes? While I turn off the use_stitch command, I can get the result soon...Hoping that u can help me with that ：）

**Comment by wadmes** [comment]
Do you also enable gen_stitch?

**Comment by Wendyuf** [comment]
> e

yes, and it had generated the file "sim_c1_out.gds_dgSimplification.gds"，when I am debugging, it seems to be stuck in the this->solve_component(itBgn, itEnd, comp_id) process.
